_security/access-control/users-roles.md

```diff
@@ -129,14 +129,17 @@ Role | Description
 `snapshot_management_full_access` | Grants full permissions to all snapshot management actions.
 `snapshot_management_read_access` | Grants permissions to view policies but not create, modify, start, stop, or delete them.
 `point_in_time_full_access` | Grants full permissions to all Point in Time operations.
+`security_analytics_full_access` | Grants full permissions to all Security Analytics functionality.
+`security_analytics_read_access` | Grants permissions to view the various components in Security Analytics, such as detectors, alerts, and findings. It also includes permissions that allow users to search for detectors and rules. This role does not allow a user to perform actions such as modfying or deleting a detector.
+`security_analytics_ack_alerts` | Grants permissions to view and acknowledge alerts.
+
```

Comment:
This table is also missing pipe characters at the beginning and end of lines. I know you didn't add the table but like the other comment it should have those pipe characters at the beginning and end of each line per https://www.markdownguide.org/extended-syntax/#tables